### ISSUE TITLE
WaterFox android browser

### ISSUE THREAD

**Issue by sankhababu** [Tobi823/ffupdater#612]
Sir 
Is the WaterFox browser for android worthy to be included in FF-updater?
If you think right, please..
Regards.

**Comment by rafnov** [comment]
I am using Waterfox on desktop pcs and mobiles. I am disappointed that it isn't included in Ffupdater.

**Comment by Tobi823** [comment]
Do you know from where the APK files can be downloaded? 

https://github.com/BrowserWorks/Waterfox-Android does not have any releases and https://www.waterfox.net/download/ does not have any Android downloads.

**Comment by Tobi823** [comment]
@rafnov I'm sure that this was not your intention but:

>  I am disappointed

sounds disrespectful and insulting to me. I'm not obligated to support every browser in FFUpdater. It is still my free time that I invest in this app.

For the next time - please write it more positive like: "I'm a big fan of Waterfox and would love to see it in FFUpdater."

Have a nice weekend

**Comment by sankhababu** [comment]
> Do you know from where the APK files can be downloaded?
> 
> https://github.com/BrowserWorks/Waterfox-Android does not have any releases and https://www.waterfox.net/download/ does not have any Android downloads.

It's only available in play store with 'play app signing'.
There are many issues raised already in their Github page but without any response from the devs. 
Do you think something lack of transparency is there in android built due to Google's intervention?

**Comment by rafnov** [comment]
> @rafnov I'm sure that this was not your intention but:
> 
> >  I am disappointed
> 
> sounds disrespectful to me. I'm not obligated to support every browser in FFUpdater. It is still my free time that I invest in this app.
> 
> For the next time - please write it more positive like: "I'm a big fan of Waterfox and would love to see it in FFUpdater."
> 
> Have a nice weekend

I am sorry if you felt offended, that was not my intention indeed. Just please keep in mind language differences, words may have different meanings, depending on context too.

**Comment by Tobi823** [comment]
> Do you think something lack of transparency is there in android built due to Google's intervention?

I don't think so. As far as I understand "Play App Signing" does not impact the developers in publishing updates. 

> There are many issues raised already in their Github page but without any response from the devs.

It is a bit strange - but this is more problematic: The last published code change (Git commit) was made on 19.04.2024. But according to the Play Store, Waterfox was updated on 13.06.2024. Maybe they were to busy and forgot to upload the latest code changes to GitHub.

I think it is best to wait for https://github.com/BrowserWorks/Waterfox-Android/issues/64

**Comment by rafnov** [comment]
They are SO busy that totally ignore user requests. I don't think Waterfox Android is worthy attention anymore as they seem ignored latest Mozilla security issue: https://www.mozilla.org/en-US/security/advisories/mfsa2024-51/ announced October 9, 2024, when latest version of their app is September 18. 
That's not all: their autofill service broke suddenly over a month ago and has't been fixed yet. 
Something wrong is going on there.

**Comment by sankhababu** [comment]
@Tobi823 

Sir 
Waterfox released apk in Github.

https://github.com/BrowserWorks/waterfox-android/releases/tag/1.1.0

Could you please take a look?

**Comment by rafnov** [comment]
There is 1.1.1 already
Apks are there.
